tailwind.config.js

```diff
@@ -32,8 +33,9 @@ module.exports = {
       textLight: '#6D6986',
       borderLight: '#CBC9D9',
 
+      lightBlue: '#2E57E8',
       darkBlue: '#6A93FF',
-      lightBlue: 'rgba(106, 147, 255, 0.5)',
+      //lightBlue: 'rgba(106, 147, 255, 0.5)',
```

Comment:
Is this commented as you want to revert it back later?

Reply:
It's commented because I previously used that lightBlue somewhere but have now overriden it, but am not yet sure what the other one should be, so I've left this here as a reference to work out. I'll add a TODO.

```suggestion
      //TODO: Work out where this was used and what it should have been
      //lightBlue: 'rgba(106, 147, 255, 0.5)',
```